src/main/java/it/gov/pagopa/debtposition/mapper/ConvertPPModelToPPEntityForUpdate.java

```diff
@@ -1,117 +1,90 @@
+
+    destination.setPayStandIn(source.isPayStandIn());
+    destination.setCity(source.getCity());
+    destination.setCivicNumber(source.getCivicNumber());
+    destination.setCompanyName(source.getCompanyName());
+    destination.setCountry(source.getCountry());
+    destination.setEmail(source.getEmail());
+    destination.setFiscalCode(source.getFiscalCode());
+    destination.setFullName(source.getFullName());
+    destination.setIupd(source.getIupd());
+    destination.setOfficeName(source.getOfficeName());
+    destination.setPhone(source.getPhone());
+    destination.setPostalCode(source.getPostalCode());
+    destination.setProvince(source.getProvince());
+    destination.setRegion(source.getRegion());
+    destination.setStreetName(source.getStreetName());
+    destination.setType(source.getType());
+    destination.setValidityDate(source.getValidityDate());
+    destination.setSwitchToExpired(
+        null != source.getSwitchToExpired() && source.getSwitchToExpired());
+
+    List<PaymentOptionModel> paymentOpts = source.getPaymentOption();
+    if (null != paymentOpts && !paymentOpts.isEmpty()) {
+      for (PaymentOptionModel pOModel : paymentOpts) {
+        destination.addPaymentOption(this.convertPOModelToPOEntityForUpdate(pOModel));
+      }
     }
 
-    private Transfer convertTransfModelToTransfEntityForUpdate(TransferModel tm) {
-
-        Transfer t = new Transfer();
-        t.setAmount(tm.getAmount());
-        t.setOrganizationFiscalCode(tm.getOrganizationFiscalCode());
-        t.setCompanyName(tm.getCompanyName());
-        t.setCategory(tm.getCategory());
-        t.setIban(tm.getIban());
-        t.setIdTransfer(tm.getIdTransfer());
-        t.setPostalIban(tm.getPostalIban());
-        if (tm.getStamp() != null) {
-            t.setHashDocument(tm.getStamp().getHashDocument());
-            t.setStampType(tm.getStamp().getStampType());
-            t.setProvincialResidence(tm.getStamp().getProvincialResidence());
-        }
-        t.setRemittanceInformation(tm.getRemittanceInformation());
-        
-        List<TransferMetadataModel> metadata = tm.getTransferMetadata();
-        if (!CollectionUtils.isEmpty(metadata)) {
-            for (TransferMetadataModel m : metadata) {
-                t.addTransferMetadata(TransferMetadata.builder().key(m.getKey()).value(m.getValue()).build());
-            }
-        }
-
-        return t;
+    return destination;
+  }
+
+  private PaymentOption convertPOModelToPOEntityForUpdate(PaymentOptionModel pom) {
+
+    PaymentOption po = new PaymentOption();
+    po.setAmount(pom.getAmount());
+    po.setDescription(pom.getDescription());
+    po.setDueDate(pom.getDueDate());
+    po.setFee(pom.getFee());
+    po.setIsPartialPayment(pom.getIsPartialPayment());
+    po.setIuv(pom.getIuv());
+    po.setNav(pom.getNav());
+    po.setRetentionDate(pom.getRetentionDate());
+    po.setNotificationFee(pom.getNotificationFee());
+    // set debtor fields default values for v1 model
```

Comment:
could be used payment position debtor?

Reply:
@cap-ang what is your doubt ?

Reply:
the doubt was whether to assign, for v1 conversions, to all new debtor fields of the `payment_option` the debtor input values in the `payment_position`